src/main/java/com/dougnoel/sentinel/steps/APISteps.java

```diff
@@ -0,0 +1,173 @@
+package com.dougnoel.sentinel.steps;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.dougnoel.sentinel.apis.APIManager;
+import com.dougnoel.sentinel.apis.Response;
+import com.dougnoel.sentinel.configurations.Configuration;
+import com.dougnoel.sentinel.enums.RequestType;
+import com.dougnoel.sentinel.strings.SentinelStringUtils;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class APISteps {
+	private static final Logger log = LogManager.getLogger(APISteps.class.getName()); // Create a logger.
+	
+	/**
+	 * Loads an API based on the environment you are currently testing.
+     * Refer to the documentation in the sentinel.example project for more information. 
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>I use the API named Agify API</li>
+     * <li>I use the API named My API</li>
+     * </ul>
+     * <p>
+     * 
+	 * @param apiName name of the API object we want to use
+	 */
+	@Given("^I use the API named (.*?)$")
+	public void setAPI(String apiName) {
+        APIManager.setAPI(apiName);
+	}
+
+	/**
+	 * Sets the body of the active API call to the string passed.
+	 * 
+	 * @param body String the json to be passed as the body of the request.
+	 */
+	@Given("I set the API body to")
```

Comment:
Missing capture group, no?

Reply:
Also, does it make more sense to say we are setting the "request body", rather than the "API body"?

Reply:
Good call in the naming. Fixed another step as well. We aren't missing a capture group. You'll notice this isn't surrounded by a ^$. We are actually using this as a cucumber step that doesn't use regex, and so it expects an argument through some magic I do not understand. It works however.